src/components/grid.jsx

```diff
@@ -146,6 +148,27 @@ function getOpacity(snake, highlightedSnake) {
   return snake.isDead ? DEAD_OPACITY : 1;
 }
 
+function getPartOpacity(part) {
```

Comment:
For chained if/else statements, I like to see a `switch` used instead.

Reply:
TBH that's a bit of stylistic feedback.  I personally have never been a fan of switch stmts if there is only a single option and "default"